include/SSVOpenHexagon/Data/StyleData.hpp

```diff
@@ -33,8 +34,12 @@ class StyleData
     float pulseIncrement;
     bool huePingPong;
     float maxSwapTime;
+    int colorPosOffset;
+    float bgTileRadius;
+    float bgRotOff;
     float _3dDepth;
     float _3dSkew;
+    sf::Vector2f skew;
```

Comment:
This one should be fine because the default constructor of `sf::Vector2f` zeroes the values out (hopefully).